fluent-bit/fluent-bit-go-loki/version.go

```diff
@@ -0,0 +1,3 @@
+package main
+
+const Version = "0.1.3"
```

Comment:
Would be nice if you could tag this the same way we tag Loki binaries. (eg see `GO_FLAGS` in our makefile.)

Reply:
You means that using build-time variable to specify plugin version, right?

But other option which is `-a -tags netgo -installsuffix netgo --ldflags '-extldflags "-static"'` is unusable for building plugin because it is not executables but shared object:

```log
% LANG=C go build -a -tags netgo -installsuffix netgo --ldflags '-extldflags "-static"' -buildmode=c-shared -o out_loki.so .
# github.com/cosmo0920/fluent-bit-go-loki
/home/hhatake/.goenv/versions/1.12.7/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
/usr/bin/ld: /usr/lib/gcc/x86_64-linux-gnu/6/crtbeginT.o: relocation R_X86_64_32 against hidden symbol `__TMC_END__' can not be used when making a shared object
/usr/bin/ld: final link failed: Nonrepresentable section on output
collect2: error: ld returned 1 exit status
```

Reply:
I've fixed with https://github.com/grafana/loki/pull/847/commits/c18bc38813c2c1d1e8996fde1da1f0e20c67222e.